docs/release-process.md

```diff
@@ -0,0 +1,63 @@
+# Release Process
+
+## Web wallet
+
+Staging:
+- checkout master branch
+- build
+- deploy to <https://wallet.stg.oasisprotocol.org/>
+
+Production:
+- ensure <https://wallet.stg.oasisprotocol.org/> works
+  - especially features related to changes
+  - look at CSP errors (expect one error: blocked `eval` in `inquire`)
+- see the footer for what commit is deployed
+- update stable branch to that commit
+- build
+- deploy to <https://wallet.oasisprotocol.org/>
+
+### Build
+
+```sh
+yarn install --frozen-lockfile
+yarn build  # oasisscan
```

Comment:
clarify `# oasisscan`

meaning that it builds the oasisscan variant?
meaning you sometimes use `oasisscan` as a command line arg but commented it out here?